components/breadcrumbs/src/HeadlessBreadcrumbs.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { CSSProperties } from "react";
+
+interface HeadlessBreadcrumbsInterface {
+	breadcrumbs: (React.ReactNode | string)[];
+	separator: React.ReactNode | string;
+	breadcrumbsClasses?: string | CSSProperties;
+	breadcrumbsContainerClasses?: string | CSSProperties;
+	separatorClasses?: string | CSSProperties;
+}
+
+const HeadlessBreadcrumbs = ({
+	breadcrumbs,
+	separator,
+	breadcrumbsClasses,
+	breadcrumbsContainerClasses,
+	separatorClasses,
+}: HeadlessBreadcrumbsInterface) => {
+	return (
+		<div
+			className={
+				typeof breadcrumbsClasses === "string" ? breadcrumbsClasses : ""
+			}
+			style={typeof breadcrumbsClasses === "object" ? breadcrumbsClasses : {}}
+		>
+			{breadcrumbs.map((item: React.ReactNode, index: number) => {
+				return (
+					<div
+						key={index}
+						className={
+							typeof breadcrumbsContainerClasses === "string"
+								? breadcrumbsContainerClasses
+								: ""
+						}
+						style={
+							typeof breadcrumbsContainerClasses === "object"
+								? breadcrumbsContainerClasses
+								: {}
+						}
+					>
+						{item}{" "}
```

Comment:
Is this { " " } needed? Can we add gap-x-1 or gap-x-2 to give eual spacing between all elements?

Reply:
it is auto added when save it no added because separate classes is given for styles